chrome-launcher/chrome-launcher.ts

```diff
@@ -171,6 +166,11 @@ export class Launcher {
       this.chromePath = installations[0];
     }
 
+    // Kill spawned Chrome process in case of ctrl-C.
+    if (defaults(this.opts.handleSIGINT, true)) {
+      process.on(_SIGINT, this.sigintListener.bind(this));
```

Comment:
Is this actually going to be removed later? Bind will return a new object that you're not saving anywhere (you're just removing the class-wide `this.sigintListener`) Maybe we should just keep the anonymous inline arrow func and save to `this._sigintListener`